market_nextdoor_api/views/preorder_views.py

```diff
@@ -62,4 +62,38 @@ def update_preorder(preorder, data):
 
 def delete_preorder(preorder):
   preorder.delete()
-  return Response(status=status.HTTP_204_NO_CONTENT)
\ No newline at end of file
+  return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+#manytomany views testing
+@api_view(['GET', 'POST'])
+def preorder_test_list(request, customer_id):
+  try:
+    check_customer = Customer.objects.get(pk=customer_id)
+  except Customer.DoesNotExist:
+    return Response(status=status.HTTP_404_NOT_FOUND)
+
+  if request.method == 'GET':
+    return get_preorder_test_list(request, check_customer)
```

Comment:
is the check_customer argument representing customer id that will be passed from FE?

Reply:
In this GET request, FE will be sending the customer id through the URL: '/customers/1/preorders/' - check customer finds the Customer object based on the ID passed in the request:
"try:
    check_customer = Customer.objects.get(pk=customer_id)"

Reply:
I guess I just saw that as an argument and not a method. So the argument is a method? very interesting.